abcFinance/account.py

```diff
@@ -8,7 +8,7 @@
 from enum import Enum
 
 
-class s(Enum):
+class AccountType(Enum):
```

Comment:
`s` is hard to search and may often clash with other local variables. I replaced it with `AccountType` because it is the naming used in ESL. Though another more appropriate naming is probably `AccountSide` instead of `Side`. @7brunner what is your suggestion here?

Reply:
Changed `AccountType` to `AccountSide` because ESL's `AccountType` actually consists of
```
AccountType = enum(ASSET=1,
                   LIABILITY=2,
                   INCOME=4,
                   EXPENSES=5,
                   GOOD=6)
```